src/App.js

```diff
@@ -14,11 +14,47 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    selectedGood: 'Jam',
+  };
+
+  render() {
+    const { selectedGood } = this.state;
+
+    return (
+      <div className="app">
+        {selectedGood ? (
+          <h1>
+            {`Selected good: ${selectedGood} is selected `}
+            <button
+              type="button"
+              onClick={() => {
+                this.setState({ selectedGood: null });
+              }}
+            >
+              X
+            </button>
+          </h1>
+        ) : (
+          <h1>Selected good: No goods selected</h1>
+        )}
+
+        {goodsFromServer.map(good => (
+          <button
+            key={good}
+            type="button"
+            className={selectedGood === good ? 'active' : ''}
+            onClick={() => {
+              this.setState({ selectedGood: good });
+            }}
+          >
+            {good}
+          </button>
```

Comment:
You should make a list and then add a button to each list item so check the Task again